<launch>

<!--- Sim Time -->
<param name="/use_sim_time" value="true" />
<param name="setting_file" value="$(find lidar_slam)/data/setting.txt" />

<!--- Run Rviz-->
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_slam)/launch/test.rviz" />
<!--<arg name="map_file" default="$(find lidar_slam)/launch/traversability_map.yaml"/>-->
<!--<node pkg="map_server" type="map_server" name="map_server" args="$(arg map_file)" output="screen" />-->

<!--- lidar_slam -->
<node pkg="lidar_slam" type="gicp_compus"    name="gicp_compus"    output="screen"/>
<!---    <node pkg="lidar_slam" type="loop_detect" name="loop_detect" output="screen"/>-->
<!---    <node pkg="lidar_slam" type="map_optmization" name="map_optmization" output="screen"/>-->
<!-- <node pkg="lidar_slam" type="global_localization" name="global_localization" output="screen"/>-->

</launch>
